.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,32 @@
+name: Unit Tests
+on: [workflow_call]
+jobs:
+  acceptance-tests-labels:
+    name: Unit Tests
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-2019, windows-latest]
+    runs-on: ${{matrix.os}}
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - name: Checkout repository
+        if: ${{ runner.os == 'Linux' }}
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+      - name: Checkout repository
+        if: ${{ runner.os == 'Windows' }}
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 1
+          submodules: true
+      - name: DiskANN Build CLI Applications
+        uses: ./.github/actions/build   
+        
+      - name: Run Unit Tests
```

Comment:
![image](https://github.com/microsoft/DiskANN/assets/15103725/c710a17b-235f-479b-8641-4c12574b9263)

Reply:
this shows that the unit tests are ran. I only add one test case, so here only shows 1 test

Reply:
this shows that the unit test project is ran. I only add one test case, so here only shows 1 test